src/linker/configuration.py

```diff
@@ -85,3 +85,12 @@ def _load_input_data_paths(input_data: Path) -> List[Path]:
         if missing:
             raise RuntimeError(f"Cannot find input data: {missing}")
         return file_list
+
+    @staticmethod
+    def _get_spark_requests(
+        environment: Dict[str, Union[Dict, str]]
+    ) -> Optional[Dict[str, Any]]:
+        spark = environment.get("spark", None)
```

Comment:
Shouldn't the default be `{}` here? If not, the typing of the method should change.

Reply:
Either works. But isn't the typing correct since it's `Optional`?